third_party/NordicSemiconductor/README.md

```diff
@@ -10,3 +10,11 @@ Directory consists of following folders:
  - /nrfx         - Standalone drivers for peripherals present in Nordic SoCs (https://github.com/NordicSemiconductor/nrfx)
  - /segger_rtt   - Library for the RTT communication
  - /softdevice   - SoftDevice s140 headers
+
+ The following changes comparing to the nRF5 SDK 15.2 have been incorporated:
+ - move definitions of CDC_ACM_COMM_INTERFACE and CDC_ACM_DATA_INTERFACE from usb-cdc-uart.c to platform-config.h,
+ - change name of NRF_DFU_TRIGGER_USB_INTERFACE_NUM to NRF_DFU_TRIGGER_USB_INTERFACE,
+ - add nrf_dfu_trigger_usb_init function call wrapped in #if NRF_MODULE_ENABLED(APP_USBD_NRF_DFU_TRIGGER) to the nrf5UartInit function
+ - added #define LOG_BACKEND 0 in nrf_log_ctrl.h
```

Comment:
Let's make one bullet point from this one and two below.

What about: - modified nrf_log_ctrl.h file in order to remove unused backend logging functions.